content/en/opentelemetry/collector_exporter/configuration.md

```diff
@@ -9,13 +9,238 @@ further_reading:
   text: "OpenTelemetry Collector Configuration"
 ---
 
-## OpenTelemetry Collector configuration
+The OpenTelemetry Collector receives, processes, and exports telemetry data from your applications. To send this data to Datadog, you need to configure several components within the Collector:
 
-To learn more about configuring the Collector, read the [OpenTelemetry Collector Configuration][3] documentation.
+- [Datadog Exporter][1]: Forwards trace, metric, and logs data from OpenTelemetry SDKs on to Datadog (without the Datadog Agent).
+- [Datadog Connector][29]: Calculates Trace Metrics from collected span data.
+
+## Setting up the OpenTelemetry Collector
+
+To run the OpenTelemetry Collector with the Datadog Exporter and Datadog Connector:
+
+### Step 1 - Download the OpenTelemetry Collector
+
+Download the latest release of the OpenTelemetry Collector Contrib distribution, from [the project's repository][3].
+
+### Step 2 - Configure the Datadog Exporter and Connector
+
+To use the Datadog Exporter, add it to your [OpenTelemetry Collector configuration][4]. Create a configuration file and name it `collector.yaml`. Use the example file which provides a basic configuration that is ready to use after you set your Datadog API key as the `DD_API_KEY` environment variable:
+
+{{< code-block lang="yaml" filename="collector.yaml" collapsible="true" >}}
+receivers:
+  otlp:
+    protocols:
+      http:
+        endpoint: 0.0.0.0:4318
+      grpc:
+        endpoint: 0.0.0.0:4317
+  # The hostmetrics receiver is required to get correct infrastructure metrics in Datadog.
+  hostmetrics:
+    collection_interval: 10s
+    scrapers:
+      paging:
+        metrics:
+          system.paging.utilization:
+            enabled: true
+      cpu:
+        metrics:
+          system.cpu.utilization:
+            enabled: true
+      disk:
+      filesystem:
+        metrics:
+          system.filesystem.utilization:
+            enabled: true
+      load:
+      memory:
+      network:
+      processes:
+  # The prometheus receiver scrapes metrics needed for the OpenTelemetry Collector Dashboard.
+  prometheus:
+    config:
+      scrape_configs:
+      - job_name: 'otelcol'
+        scrape_interval: 10s
+        static_configs:
+        - targets: ['0.0.0.0:8888']
+
+  filelog:
+    include_file_path: true
+    poll_interval: 500ms
+    include:
+      - /var/log/**/*example*/*.log
+
+processors:
+  batch:
+    send_batch_max_size: 100
+    send_batch_size: 10
+    timeout: 10s
+
+connectors:
+  datadog/connector:
+
+exporters:
+  datadog/exporter:
+    api:
+      site: <DD_SITE>
```

Comment:
Would it work if we just do this?
```suggestion
      site: {{< region-param key="dd_site">}}
```

Reply:
Good thinking! Swapped to use the dynamic site. I just had to change the code block to use a typical code fence instead of the code shortcode so it renders properly.